src/utils/Paths.ts

```diff
@@ -10,6 +10,7 @@ export const HEALTH_CHECK_URI = `${ROOT_URI}/healthcheck`;
 // Pages
 export const ENTRY_PAGE_URI = `${ROOT_URI}/start`;
 export const PENALTY_DETAILS_PAGE_URI = `${ROOT_URI}/penalty-reference`;
+export const SELECT_YEAR_PAGE_URI = `${ROOT_URI}/select-the-year`;
```

Comment:
should the path be `/select-the-penalty`?

Reply:
yes oops

Reply:
please also update the const name